tests/test_cloudformation/test_cloudformation_stack_crud.py

```diff
@@ -188,6 +224,14 @@ def test_describe_stack_by_stack_id():
     stack_by_id.stack_name.should.equal("test_stack")
 
 
+@mock_cloudformation_deprecated
```

Comment:
I should have mentioned this before - please add a test for boto3 as well (`@mock_cloudformation`)

The test could do with a few asserts as well - please verify that the table exists first, but no longer exist after deletion